codegen/runner/runner.go

```diff
@@ -59,6 +59,23 @@ func main() {
 		err, fmt.Sprintf("cant get abs path of config dir %s", configDirName),
 	)
 
+	packageHelper, err := codegen.NewPackageHelper(
```

Comment:
can pass this into GatewaySpec constructor.

Creates less confusion refactors etc, for example zhenghui has a pending PR that refactors the `NewPackageHelper()` constructor.